documentation/release_notes.rst

```diff
@@ -8,6 +8,122 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 creating efficient heterogeneous applications.
 
+New in 2022.7.0
+===============
+News
+------------
+
+New Features
+------------
+- Improved performance of the ``adjacent_find``, ``all_of``, ``any_of``, ``copy_if``, ``exclusive_scan``, ``equal``,
+``find``, ``find_if``, ``find_end``, ``find_first_of``, ``find_if_not``, ``inclusive_scan``, ``includes``, ``is_heap``,
+``is_heap_until``, ``is_partitioned``, ``is_sorted``, ``is_sorted_until``, ``lexicographical_compare``, ``max_element``,
+``min_element``, ``minmax_element``, ``mismatch``, ``none_of``, ``partition``, ``partition_copy``, ``reduce``,
+``remove``, ``remove_copy``, ``remove_copy_if``, ``remove_if``, ``search``, ``search_n`` ``stable_partition``,
+``transform_exclusive_scan``, ``transform_inclusive_scan``, ``unique``, and ``unique_copy`` algorithms with device
+policies. 
+- Improved performance of ``sort``, ``stable_sort`` and ``sort_by_key`` algorithms on GPU devices when using Merge
+sort [#fnote1]_.
+- Added ``stable_sort_by_key`` algorithm in `namespace oneapi::dpl`. 
+- In `namespace oneapi::dpl::ranges` added the range-based versions of the following algorithms: ``all_of``, ``any_of``,
+``none_of``, ``for_each``, ``find``, ``find_if``, ``find_if_not``, ``adjacent_find``, ``search``, ``search_n``,
+``transform``, ``sort``, ``stable_sort``, ``is_sorted``, ``merge``, ``count``, ``count_if``, ``equal``, ``copy``,
+``copy_if``, ``min_element``, ``max_element``.
+- Added support for operators ==, !=, << and >> for RNG engines and distributions.
+- Added experimental support for the philox RNG engine in `namespace oneapi::dpl::experimental`.
+
+Fixed Issues
+------------
+- Fixed unused variable and unused type warnings.
+- Fixed memory leaks fix when using ``sort`` and ``stable_sort`` algorithms with the oneTBB backend.
+- Fixed a build error with ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` in combination with MSVC STL and with C++20
+enabled.
+- Fixed an issue with ``histogram`` algorithm, reordering template parameters to match function parameter order. Users
+are encouraged to remove explicit specification of template parameters where they may exist from their histogram calls
+and instead add any explicit type conversions of their arguments as required.
+- ``radix_sort`` and ``radix_sort_by_key`` kernel templates residing in the
+``oneapi::dpl::experimental::kt::gpu::esimd`` now throw ``std::bad_alloc`` if they fail to allocate global memory.
+- Fixed a potential hanging occurring with ``radix_sort`` and ``radix_sort_by_key`` kernel templates residing in
+  the ``oneapi::dpl::experimental::kt::gpu::esimd`` namespace. 
+- Fixed documentation for ``sort_by_key`` algorithm, which used to be mistakenly described as stable, despite being
+unstable for host execution policies (``seq``, ``unseq``, ``par``, ``par_unseq``). It may become unstable for device
+execution policies in the future. If stability is required, use ``stable_sort_by_key`` instead. 
```

Comment:
Rendering fixes:

```suggestion
  unstable for host execution policies (``seq``, ``unseq``, ``par``, ``par_unseq``). It may become unstable for device
  execution policies in the future. If stability is required, use ``stable_sort_by_key`` instead. 
```

Reply:
The whole "New in This Release" section needs the same treatment.